ccp/src/main/java/com/togitech/ccp/component/TogiCountryCodePicker.kt

```diff
@@ -119,9 +122,11 @@ fun TogiCountryCodePicker(
         if (initialCountryPhoneCode?.startsWith("+") != true) {
             Log.e(TAG, "initialCountryPhoneCode must start with +")
         }
-        val initialCountry: CountryData? = initialCountryPhoneCode?.let {
+        val initialCountry: CountryData? = CountryData.entries.firstOrNull {
```

Comment:
I think `initialCountryPhoneCode` is more reliable to choose initial country than `initialCountryIsoCode`